src/core/lib/config/core_configuration.h

```diff
@@ -86,6 +93,8 @@ class CoreConfiguration {
   template <typename BuildFunc>
   static void BuildSpecialConfiguration(BuildFunc build) {
     // Build bespoke configuration
+    // We don't care about the builder lock here, since it's expected this call
```

Comment:
I think this comment is no longer needed.

Reply:
fixed